src/transformStateWithClones.js

```diff
@@ -7,7 +7,36 @@
  * @return {Object[]}
  */
 function transformStateWithClones(state, actions) {
-  // write code here
+  const stateClone = { ...state };
+  const stateWithClones = [];
+
+  for (const action of actions) {
+    if (action['type'] === 'addProperties') {
```

Comment:
There's no need to use additional [] and ''. It'll work like this also: 
`action.type`